tests/modeladmin/test_checks.py

```diff
@@ -639,6 +639,18 @@ class TestModelAdmin(ModelAdmin):
             'admin.E115'
         )
 
+    def test_not_filter_again_again_again(self):
+        def get_filter():
+            pass
+
+        class TestModelAdmin(ModelAdmin):
+            list_filter = (get_filter,)
+
+        self.assertIsInvalid(
+            TestModelAdmin, ValidationTestModel,
+            "The value of 'list_filter[0]' must inherit from 'ListFilter'.",
+            'admin.E113')
```

Comment:
Please put the closing ) on the next line as in other tests.

Reply:
This was a side-effect of copy-and-pasting from line 604-607 which has it on the same line. Do you want me to fix that one as well for consistency?

Reply:
No, not necessary.